integration-tests/tests.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+
+import subprocess, argparse, shlex, sys, re, os
+from time import sleep
+
+run_options = ['put','get','exist']
+
+def run_tests(behaviour, args):
+    try:
+        print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+        p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}')).decode('ascii')
+        print(str(p))
+        if behaviour in {'put', 'get'}:
+            if f'{behaviour} errors:' in str(p) or f'panicked' in str(p):
+                sys.exit(f'{behaviour} test failed, see output')
+            if behaviour == 'put':
+                sleep(30)
```

Comment:
1. This function is not a good place to add any delays. In terms of code structure, this function is responsible for running a single test step. It is better to add delay at the higher level
2. 30 seconds is too much. Why this amount was chosen?

Reply:
This is outdated sleep for testing the inconsistent get/exist results. I will remove it.